src/readability.ts

```diff
@@ -374,6 +386,7 @@ export function preprocessMarkdown(markdown: string) {
         .use(addPeriodsToListItems)
         .use(removeJsItems)
         .use(removeUnwantedNodeTypes)
+        .use(removeHorizontalRules)
```

Comment:
@noon-dawg Ah wait sorry, you'll see this breaks the existing YAML front matter unit test. 

Likely you can swap the order so `removeHorizontalRules` plugin is used after `removeFrontmatter`. 

Let me know if that doesn't work and we can think of another approach :)

Reply:
Thank works!! Thank you 🙏